src/AppBundle/Command/RemoveImageCommand.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace AppBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\PropertyAccess\PropertyAccessor;
+
+class RemoveImageCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('app:remove-image')
+            ->setDescription('Remove image that not relation to media')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $mm = $this
+            ->getContainer()
+            ->get('sonata.media.manager.media')
+        ;
+
+        $em = $this
```

Comment:
I think you can to delete MediaObject without EntityManager - you need only MediaManager - use it.